kiwi/constants.py

```diff
@@ -83,5 +83,7 @@
 DATAFILE = 'dataset.torch'
 OPTIMIZER = 'optim.torch'
 BEST_MODEL_FILE = 'best_model.torch'
+LAST_MODEL_FOLDER = 'latest_epoch'
```

Comment:
Maybe call the variable `latest_checkpoint_folder` for consistency with the name (latest_epoch), and to show that its a checkoint not only a model?

Reply:
100% agree